### ISSUE TITLE
support different forks of airflow

### ISSUE THREAD

**Issue by marengaz** [kaxil/airflowctl#7]
### use case
all the dags we write for use on gcp composer [composer lineage](https://github.com/GoogleCloudPlatform/composer-airflow/blob/2.5.1/airflow/composer/data_lineage/entities.py#L20), so probably wouldnt render in vanilla apache airflow.

### description
google store their composer code in this repo: https://github.com/GoogleCloudPlatform/composer-airflow 
the branch name designates which version of airlfow+composer to install: https://github.com/GoogleCloudPlatform/composer-airflow/branches/all
these are not published to public pip

### possible implementation
on `init` offer the options to specify:
- airflow fork repo github url & branch name
- or maybe a local path to the airflow code?

**Comment by kaxil** [comment]
Fixed in 0.2.6 -- `pip install airflowctl==0.2.6`

**Comment by kaxil** [comment]
Fixed in 0.2.6 -- `pip install airflowctl==0.2.6`

Run the following:

```
❯ airflowctl init --airflow-version /Users/kaxilnaik/GitHub/apache/airflow --build-start my_airflow_project
```


You can pass the following env vars too:

- `AIRFLOWCTL_CONSTRAINTS`: To specify a different constraints file (Default is none for installing Airflow via source code; and is official constraints file if an Airflow version is specified)
- `AIRFLOWCTL_PIP_FLAGS`: for additional flags like `-e`
